Formula/muon.rb

```diff
@@ -0,0 +1,38 @@
+class Muon < Formula
+  desc "Meson-compatible build system"
+  homepage "https://muon.build"
+  url "https://git.sr.ht/~lattis/muon/archive/0.1.0.tar.gz"
+  sha256 "9d3825c2d562f8f8ad96d1f02167e89c4e84973decf205d127efd9293d7da35b"
+  license "GPL-3.0-only"
+  head "https://git.sr.ht/~lattis/muon", branch: "master"
+
+  depends_on "ninja"
+  depends_on "pkg-config"
+
+  def install
+    system "./bootstrap.sh", "build"
+    system "./build/muon", "setup", "-Dprefix=#{prefix}", "build"
+    system "ninja", "-C", "build"
```

Comment:
```suggestion
    system "./build/muon", "setup", "build", *std_meson_args
    system "ninja", "-C", "build", "--verbose"
```

Reply:
Muon is source-compatible with Meson but the command line interface is not. For this reason, we cannot use `std_meson_args`